### PR TITLE
Api fixes

### PR DESCRIPTION
**Pull request by sipec**

### CODE REVIEW

File: web/.eslintrc.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
     'plugin:@next/next/recommended',
+    'prettier',
```
Comment (jahooma) [inline web/.eslintrc.js:8]: Nice miscellaneous change! Could just be a separate commit to main.

File: web/pages/api/v0/_types.ts [hunk 2]
```diff
@@ -109,6 +119,10 @@ export function toLiteMarket(contract: Contract): LiteMarket {
         : closeTime,
     question,
     description,
+    textDescription:
+      typeof description === 'string'
+        ? description
+        : richTextToString(description),
```
Comment (jahooma) [inline web/pages/api/v0/_types.ts:125]: Nice!

**Review by jahooma — APPROVED** [review]
Seems like all good changes for the API. Thanks!

File: web/.eslintrc.js (hunk 1 above)
Comment (sipec) [reply web/.eslintrc.js:8]: actually this doesn't work :(

**Comment by sipec** [comment]
trying to test this before I merge but deploys are failing
